### ISSUE TITLE
Training accuracy problem

### ISSUE THREAD

**Issue by zhaijianyang** [open-mmlab/mmaction2#687]
I want to train the hmdb51 dataset on the i3d model and fine-tune it on your kinetics checkpoint, but the accuracy is only about 0.53, and the parameters are the same as the kinetics dataset. May I ask how to train to achieve the accuracy in the paper.

**Comment by irvingzhang0512** [comment]
Could you provide more details(e.g. commends/configs)

**Comment by zhaijianyang** [comment]
The command and configuration information are the same as kinetics400, except that the checkpoint of kinetics400 is loaded.I was wrong just now, the accuracy is 0.62.

**Comment by irvingzhang0512** [comment]
Did you follow this [tutorial](https://mmaction2.readthedocs.io/en/latest/tutorials/2_finetune.html) to finetune models?
Generally speaking, finetune models requires smaller learning rate.
